db/articles.js

```diff
@@ -0,0 +1,16 @@
+const collection = [];
+
+function all() {
+  return collection;
+}
+
+function add(obj, productId) {
+  collection.push(obj);
+  collection[productId - 1].id = productId;
```

Comment:
If for some reason `collection[productId - 1]` is out of bounds or not an object this will throw an error. Might want to put in a check that an objects exists in that array position.